### PR TITLE
chore: update babel config

### PR DESCRIPTION
**Pull request by edleeks87**
### Proposed behaviour

The new config should allow the `lib` directory to be built without extending the config in `carbon-factory`

### Current behaviour

Currently the building of the `lib` relies on extending the config from `carbon-factory`

### Checklist

- [x] Release notes (Conventional Commits) 
 <del>- [ ] Unit tests <del>
 <del>- [ ] Cypress automation tests <del>
 <del>- [ ] Storybook added or updated <del>
 <del>- [ ] Theme support <del>
 <del>- [ ] Typescript `d.ts` file added or updated <del>

### Additional context

A step in removing `carbon-factory` dependency. The required babel dependencies will be added when `carbon-factory` is removed

### Testing instructions

The build will fail if it does not transpile the code

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
-
 module.exports = {
-  extends: path.resolve(process.cwd(), './node_modules/carbon-factory/babel.config.js')
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        useBuiltIns: 'usage',
+        targets: {
+          browsers: 'IE 11'
```
Comment (chrisbarber86) [inline babel.config.js]: Since we're dropping IE11 support going forwards - do we need this?

Comment (edleeks87) [reply babel.config.js]: Good point will remove

Comment (edleeks87) [reply babel.config.js]: Good point I'll remove

**Comment by jamime** [comment]
We decided to keep IE11 and remove it in a later PR with the other IE11 workarounds.

**Pull request merged**